docs/docsite/rst/conf.py

```diff
@@ -18,6 +18,11 @@
 
 import sys
 import os
+from pathlib import Path
+
+import yaml
```

Comment:
Let's stick to toml that's in stdlib since Python 3.11 and doesn't require a dependency.

Reply:
OK. I noticed you had a similar comment somewhere else and did look at toml but you need to wrap keys in double quotes, etc. The `key: value` format of yaml just seemed simpler to me and we already have yaml as a build dependency. I guess there's no point doubling down on it though. I'll convert toml to yaml.

Reply:
> I'll convert toml to yaml.

vice versa?

Reply:
Also, it can use single quotes, not just double. But they have slightly different semantics: check out literal vs. basic + multi-line options @ https://toml.io/en/.

FWIW, TOML preaches less ambiguity, while in YAML you can type in `{key: 1, key2: 3.10, key3: s3.10}` and the parser would produce `{"key": 1, "key2": 3.1, "key3": "s3.10"}`. This usually causes confusion and sets the bar high for knowing subtle nuances of the spec.

Reply:
Thanks. I'm familiar with the differences between double and single but hadn't reviewed all that detail in the TOML docs. It looks like double quotes are generally preferable and they should allow for things like wildcards and escaped characters if the need for those ever arises. I was experimenting with a [wildcard](https://documatt.com/sphinx-reredirects/usage.html#wildcard-syntax) earlier but I think it is for a slightly different scenario.

Anyway I think we should opt for double quotes. That's the default string delimiter in TOML and might come in handy down the line.